src/bidiMapper/commandProcessor.ts

```diff
@@ -55,21 +63,32 @@ export class CommandProcessor {
       }
     );
 
-    this._cdpServer.setOnMessage((messageObj) => {
-      return this._onCdpMessage(messageObj);
```

Comment:
This method was intended to inject all the message listeners logic and later to be split between different domains.

Reply:
I think we can still organize this so that different domains are separate. I'm imagining `BrowsingContextProcessor` can have a public method like `registerEventListeners(cdpClient)` that we call from here. The `registerEventListeners` method can register for whatever events the `BrowsingContextProcessor` needs, and the `CommandProcessor` doesn't need to know or care.